src/Ryujinx.Common/Logging/Logger.cs

```diff
@@ -22,6 +23,9 @@ public static class Logger
 
         public readonly struct Log
         {
+            private static readonly string _homeDir = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
+            private static readonly string _homeDirRedacted = Path.Combine(new DirectoryInfo(_homeDir).Parent.FullName, "[redacted]");
```

Comment:
```suggestion
            private static readonly string _homeDirRedacted = Path.Combine(Directory.GetParent(_homeDir).FullName, "[redacted]");
```

Not even sure if that makes any difference here. It's probably fine the way it currently is as well. I looked at `FullName` vs `FullPath` as well and can't find a difference, so I'm not sure why both of them exist.

Reply:
Changed it just to be sure